registry/emailTemplates.xml

```diff
@@ -62,4 +62,5 @@
 	<email key="SUBSCRIPTION_RENEW_INSTL" can_edit="1" can_disable="0" to_role_id="2097152"/>
 	<email key="CITATION_EDITOR_AUTHOR_QUERY" can_edit="1" can_disable="0"/>
 	<email key="REVISED_VERSION_NOTIFY" can_edit="1" can_disable="0" to_role_id="512"/>
+	<email key="STATISTICS_REPORT_NOTIFICATION" can_edit="1" can_disable="0" from_role_id="16" to_role_id="17"/>
```

Comment:
If you change who the email is sent to don't forget to change this value too. :+1: